api4/team.go

```diff
@@ -1082,3 +1086,42 @@ func updateTeamScheme(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	ReturnStatusOK(w)
 }
+
+func ifGroupsThenUsersRemoved(c *Context, w http.ResponseWriter, r *http.Request) {
+	c.RequireTeamId()
+	if c.Err != nil {
+		return
+	}
+
+	if len(c.Params.GroupIDs) < 26 {
```

Comment:
Nit 0/5: A constant would work better here than a magic number

Reply:
I agree, and I'm actually surprised that one doesn't exist yet given that this is instance number 226 of this bad habit. Perhaps I'll submit a separate PR to fix all of them at once.

Reply:
Yes, I think we should have a constant for that, but I don't see adding it here. I like the idea of a whole PR replacing all the Id lengths checks with a single constant :)